library/search_generate.cc

```diff
@@ -476,11 +476,12 @@ void train()
   Generator gen(vw_obj, nullptr); // &dict);
   for (size_t pass=1; pass<=20; pass++)
   { cerr << "===== pass " << pass << " =====" << endl;
-    run_istream(gen, "phrase-table.tr", true);
-    run_istream(gen, "phrase-table.tr", false, 300000);
-    run_istream(gen, "phrase-table.te", false, 100000);
+    //run_istream(gen, "phrase-table.tr", true);
+    //run_istream(gen, "phrase-table.tr", false, 300000);
+    //run_istream(gen, "phrase-table.te", false, 100000);
+    run_easy();
   }
-  VW::finish(vw_obj);
+  VW::finish(vw_obj, false);
```

Comment:
What calls delete now?

Reply:
It wasn't being deleted. Just updated it to delete and commented out this line `Generator gen(vw_obj, nullptr); // &dict);` which was causing a memory freeing error

Reply:
But wont calling finish with false mean it wont be deleted?
```
void finish(VW::workspace& all, bool delete_all)
```

Reply:
Yes I removed `false`